### PR TITLE
Improved parameters for Skia shadows

### PR DESCRIPTION
**Pull request by jason-simmons**

### CODE REVIEW

**Comment by jason-simmons** [comment]
@abarth

File: flow/layers/physical_model_layer.cc [hunk 1]
```diff
@@ -77,11 +77,14 @@ void PhysicalModelLayer::DrawShadow(SkCanvas* canvas, const SkPath& path,
     SkShadowFlags flags = transparentOccluder ?
         SkShadowFlags::kTransparentOccluder_ShadowFlag :
         SkShadowFlags::kNone_ShadowFlag;
+    const SkRect& bounds = path.getBounds();
+    SkScalar shadowX = (bounds.left() + bounds.right()) / 2;
+    SkScalar shadowY = bounds.top() - 600.0f;
```
Comment (abarth) [inline flow/layers/physical_model_layer.cc]: shadow_x / shadow_y

**Comment by abarth** [comment]
LGTM

**Comment by Hixie** [comment]
It's a bit weird that we hard-code these values in the engine rather than making the framework provide them, but we can change that later.

**Pull request merged**